### PR TITLE
 Fix #131: ensure absolute workspace root URI

### LINKED ISSUE #131: ServerConfiguration should ensure absolute workspace root
If the model server is launched with a relative path in the `-r` parameter, then the `ServerConfiguration` will have a relative path for its workspace root and consequently all resources that the *Model Server* manages will have relative URIs.  This will break resource set save and possibly more (EMF run-time really needs resources to have absolute URIs).

### PR DESCRIPTION
**Pull request by cdamus**
***Note** that this change builds on PR #128 for other fixes in URI computations, which should be merged before this.*

### CODE REVIEW

File: com.eclipsesource.modelserver.emf/src/test/java/com/eclipsesource/modelserver/emf/configuration/ServerConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package com.eclipsesource.modelserver.emf.configuration;
```
Comment (edgarmueller) [inline com.eclipsesource.modelserver.emf/src/test/java/com/eclipsesource/modelserver/emf/configuration/ServerConfigurationTest.java:16]: Header missing

File: com.eclipsesource.modelserver.emf/src/test/java/com/eclipsesource/modelserver/emf/configuration/ServerConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    }
+    
+	@Test
+	public void workspaceRoot() {
+		File cwd = getCWD();
+
+		serverConfiguration.setWorkspaceRootURI(URI.createFileURI(cwd.getAbsolutePath()));
+		assertThat(serverConfiguration.getWorkspaceRoot(), is(cwd.getAbsolutePath()));
+		
+		serverConfiguration.setWorkspaceRoot(".");
```
Comment (edgarmueller) [inline com.eclipsesource.modelserver.emf/src/test/java/com/eclipsesource/modelserver/emf/configuration/ServerConfigurationTest.java:57]: Could we move this into its own test?

**Review by edgarmueller — COMMENTED** [review]
LGTM! Two minor remarks

**Review by ndoschek — APPROVED** [review]
:+1:

**Pull request merged**